receiver/hostmetricsreceiver/internal/scraper/processscraper/config.go

```diff
@@ -31,10 +31,10 @@ type Config struct {
 	Include MatchConfig `mapstructure:"include"`
 	Exclude MatchConfig `mapstructure:"exclude"`
 
-	// MuteProcessNameError is a flag that will mute the error encountered when trying to read a process the
-	// collector does not have permission for.
+	// MuteProcessErrors is a flag that will mute the error encountered when trying to read a process the
+	// collector does not have permission for or try to read parent process on k8s environments which does not exist.
```

Comment:
I think k8s is an unnecessary example here 

```suggestion
	// MuteProcessErrors is a flag that will mute the errors encountered when the collector is trying to read
	// a process details that it does not have access to.
```